apps/prairielearn/src/server.ts

```diff
@@ -1280,8 +1280,8 @@ export async function initExpress(): Promise<Express> {
       .default,
   );
   app.use(
-    '/pl/course_instance/:course_instance_id(\\d+)/instructor/instance_admin/access',
-    (await import('./pages/instructorInstanceAdminAccess/instructorInstanceAdminAccess.js'))
```

Comment:
Consider adding a redirect to the old page in case anyone has bookmarks lying around (unlikely, but it's best practice to redirect when you move something).

Reply:
Discussed in meeting, this has been deemed unnecessary.